cisco_aci/datadog_checks/cisco_aci/cisco.py

```diff
@@ -152,7 +152,7 @@ def check(self, instance):
         self.service_check(SERVICE_CHECK_NAME, AgentCheck.OK, tags=service_check_tags)
 
         if set_external_tags:
-            set_external_tags(self.get_external_host_tags())
+            self.set_external_tags(self.get_external_host_tags())
```

Comment:
Should we remove the import for `set_external_tags` at the top ?

Reply:
No b/c we need a way to tell if on A5

Reply:
let's do 
```python
if hasattr(self, "set_external_tags"):
```
then

Reply:
Or just remove completely that if. We have pinned integrations in A5

Reply:
➕